### DISCUSSION TITLE
[RISOLTO] problema con zigbee2mqtt

### DISCUSSION THREAD

**Discussion by mariomare22** [indomus/forum#88] [ZigBee, Z-Wave, Thread, Matter]
Buongiorno a todos!
Da qualche giorno ho problemi a comandare le lampade sia tramite z2m che tramite home assistant.
il sensore di temperatura funziona bene; qui sotto c'e' il log di z2m.

```
Zigbee2MQTT:info  2021-04-03 13:02:13: Successfully configured 'lampada1'
Zigbee2MQTT:info  2021-04-03 13:02:13: MQTT publish: topic 'zigbee2mqtt/bridge/response/device/configure', payload '{"data":{"id":"lampada1"},"status":"ok","transaction":"g2h0x-1"}'
Zigbee2MQTT:error 2021-04-03 13:02:27: Publish 'set' 'state' to 'lampada1' failed: 'Error: Command 0x842e14fffe19609e/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
Zigbee2MQTT:info  2021-04-03 13:02:27: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'lampada1' failed: 'Error: Command 0x842e14fffe19609e/1 genOnOff.on({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null,\"writeUndiv\":false}) failed (no response received)'","meta":{"friendly_name":"lampada1"},"type":"zigbee_publish_error"}'

```

non riesco neppure ad eliminare la lampadina per poi riaggiungerla.
Avete idee?

Grazie

**Comment by snakuzzo** [comment]
Se non ti funziona da webgui di zigbee2mqtt è inutile che provi da HA :D

Quel `failed (no response received)` credo indichi che non c'è comunicazione con la lampada o che comunque la lampada non riceve nulla.

**Comment by mariomare22** [comment]
Boh staccando e ri-attaccando il conbee si è ripreso.

**Reply by snakuzzo** [reply]
queste cose però dovreste segnalarle sul progetto zigbee2mqtt su github a mio avviso.
Così come si sfruttano soluzioni, bisognerebbe contribuire in caso di problemi...almeno segnalando queste situazioni.
Ti inviterei ad aprire una issue anche lì prima cercando se c'è un problema simile e poi eventualmente segnalando questa cosa...visto soprattutto il fatto che la Conbee2 è ancora supportata in via sperimentale
